### PR TITLE
feat(oauth): remove keytar dependencies

### PR DESCRIPTION
**Pull request by olamothe**
Removed the `Storage` (OAuthStorage) module using keytar, and simply use the configuration for the access token obtained in the login flow.

https://coveord.atlassian.net/browse/CDX-193

### CODE REVIEW

File: packages/cli/src/commands/config/get.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export default class Get extends Command {
 
   async run() {
     const cfg = await new Config(this.config.configDir, this.error).get();
+    delete cfg.accessToken;
```
Comment (olamothe) [inline packages/cli/src/commands/config/get.ts:13]: Delete the access token from the config before printing it to console with `coveo config:get`

Comment (y-lakhdar) [reply packages/cli/src/commands/config/get.ts:13]: Does that mean we have to run `auth:login` after doing a `config:get` command?

Comment (olamothe) [reply packages/cli/src/commands/config/get.ts:13]: No, it just means we delete it from the object we use to display in the console.

It does not actually remove it from the config, on your disk.

**Review by y-lakhdar — APPROVED** [review]
Should the keytar dependencies be removed from the Dockerfile in this PR too?

**Review by louis-bompart — CHANGES_REQUESTED** [review]
> Should the keytar dependencies be removed from the Dockerfile in this PR too?

This and :shipit: I'd say :D

File: packages/cli-e2e/package.json [hunk 2]
```diff
@@ -20,16 +20,16 @@
     "test:headless": "node ./entrypoints/entry.js",
     "test:debug": "node ./entrypoints/entry.js --debug",
     "test:bash": "node ./entrypoints/entry.js --bash",
-    "jest": "jest",
+    "jest": "jest --verbose",
```
Comment (louis-bompart) [inline packages/cli-e2e/package.json:23]: Why verbose?

Comment (louis-bompart) [inline packages/cli-e2e/package.json:23]: Why verbose

**Review by louis-bompart — APPROVED** [review]
lgtm, not sure about verbose tho

File: packages/cli-e2e/package.json (hunk 2 above)
Comment (olamothe) [reply packages/cli-e2e/package.json:23]: It's nicer to look at :D

**Pull request merged**